error.internal=Ocorreu um erro interno. Tente novamente mais tarde.
auth.login.failed=Não foi possível autenticar o usuário {0}. Verifique suas credenciais e tente novamente.